docs/my-app-with-ember-css-modules/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # my-app-with-ember-css-modules
 
+## 1.1.0
+
+### Minor Changes
+
+- [#107](https://github.com/ijlee2/embroider-css-modules/pull/107) Deprecated the helpers {{local-class}} and {{local-class-new}} ([@ijlee2](https://github.com/ijlee2))
```

Comment:
The changes are substantial enough that I think a minor release is warranted for the docs and test apps.